### ISSUE TITLE
[Kannada] glyph and placement of subscript ṣ in ಕ್ಷ

### ISSUE THREAD

**Issue by sridatta1** [TiroTypeworks/Indigo#18]
ṣ as second consonant of a cluster, commonly occurs in the conjunct kṣa. It also occurs in ṣṣa tṣa, but the frequency of the words containing them are rare.
As an exception, in ಕ್ಷ the subscript ಷ is generally written below the consonant, not as right bottom like other ottus.
In Tiro Kannada font, ಷ ottu is in the right bottom.
![IMG_20220604_134415__01__01](https://user-images.githubusercontent.com/23263723/171991792-a4b589fc-f660-4f07-8a2a-b302fcac42ea.jpg)

![image](https://user-images.githubusercontent.com/23263723/171991393-cbf99c6b-91ea-46c2-8310-f3c191d4300f.png)
Rendering in some other fonts.
![image](https://user-images.githubusercontent.com/23263723/171991579-d25e4865-266b-464a-a89f-30947818c4e9.png)
![image](https://user-images.githubusercontent.com/23263723/171991591-0ce5ba4f-a720-466c-ba9e-ed4d8f20a39e.png)
Similarly, ಕ್ಷ as subscripts as in ತ್ಕ್ಷ needs to be corrected
![image](https://user-images.githubusercontent.com/23263723/171991663-e1c100ef-c5d3-4c09-b70d-7afc42147fde.png)
![image](https://user-images.githubusercontent.com/23263723/171991676-ef449a6a-ab0b-4edd-b5e2-db6cc6eb093d.png)
But, how the subscript needs to be written with other consonants like ಷ್ಷ ತ್ಷ needs to be examined as they are very rare.
![image](https://user-images.githubusercontent.com/23263723/171991750-58161a20-ea7d-4297-ab7a-acaca8e5d086.png)
![image](https://user-images.githubusercontent.com/23263723/171991756-2cd45a0e-dd58-42d9-b924-c3604cdd57ca.png)
With ಷ್ಷ, the current form in the font can be retained.

**Comment by tiroj** [comment]
Thanks. I’ll consider this. We do have ಕ್ಷ and related syllables as composed glyphs in the font, so revising the shape and position of the subscript would be possible with only an adjustment to these glyphs and updates to kerning of adjacent letters.
<img width="1521" alt="image" src="https://user-images.githubusercontent.com/5614500/172060764-98a378f0-484b-4637-a222-0d39b79daa75.png">

**Comment by tiroj** [comment]
Of course, if we change this, existing documents will reflow, which may be a concern for some users. Probably sooner is better than later.
